appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -3616,6 +3626,19 @@ String newerVersionComponentException(String componentType, int srcCompVersion,
   String UnitProperties();
 
   //Params
+  @DefaultMessage("scrollPosition")
+  @Description("")
+  String scrollPositionParams();
+
+  @DefaultMessage("position")
+  @Description("")
+  String positionParams();
+
+  @DefaultMessage("displacement")
```

Comment:
I think that `displacement` might be a term unknown to some of our younger users (~10 years old). I think it might be better for the related methods to use the term `amount` or `amountInPixels` (to indicate units), e.g. `ScrollBy(amount)`.